packages/studio-base/src/panels/StateTransitions/index.stories.tsx

```
@@ -122,171 +120,83 @@ export default {
-    return (
-      <PanelSetup fixture={fixture} pauseFrame={pauseFrame} includeSettings>
-        <StateTransitions
-          overrideConfig={{
-            paths: [{ value: "/some/topic/with/state.state", timestampMethod: "receiveTime" }],
-            isSynced: true,
-          }}
-        />
-      </PanelSetup>
-    );
-  },
-
-  play: async (ctx) => {
-    await ctx.parameters.storyReady;
-  },
-
-  parameters: { useReadySignal: true },
+  render: () => (
+    <PanelSetup fixture={fixture} includeSettings>
+      <StateTransitions
+        overrideConfig={{
+          paths: [{ value: "/some/topic/with/state.state", timestampMethod: "receiveTime" }],
+          isSynced: true,
+        }}
+      />
+    </PanelSetup>
+  ),
 };
 
 export const MultiplePaths: StoryObj = {
-  render: function Story() {
-    const readySignal = useReadySignal({ count: 3 });
-    const pauseFrame = useCallback(() => readySignal, [readySignal]);
-    return (
-      <PanelSetup fixture={fixture} pauseFrame={pauseFrame}>
-        <StateTransitions
-          overrideConfig={{
-            paths: new Array(5).fill({
-              value: "/some/topic/with/state.state",
-              timestampMethod: "receiveTime",
-            }),
-            isSynced: true,
-          }}
-        />
-      </PanelSetup>
-    );
-  },
-
-  play: async (ctx) => {
-    await ctx.parameters.storyReady;
-  },
-
-  parameters: { useReadySignal: true },
-};
-
-export const MultiplePathsWithHover: StoryObj = {
-  render: function Story() {
-    const readySignal = useReadySignal({ count: 3 });
-    const pauseFrame = useCallback(() => readySignal, [readySignal]);
-    return (
-      <PanelSetup
-        fixture={fixture}
-        pauseFrame={pauseFrame}
-        onMount={() => {
-          const mouseEnterContainer = document.querySelectorAll(
-            "[data-testid~=panel-mouseenter-container",
-          )[0]!;
-          TestUtils.Simulate.mouseEnter(mouseEnterContainer);
```

This was failing silently prior to this and was likely related to the old show buttons on hover behavior that we don't have anymore